pandas/tests/tools/test_to_timedelta.py

```diff
@@ -121,6 +121,25 @@ def test_to_timedelta_invalid(self):
             invalid_data, to_timedelta(invalid_data, errors="ignore")
         )
 
+    @pytest.mark.parametrize(
+        "val, warning",
+        [
+            ("1M", FutureWarning),
+            ("1 M", FutureWarning),
+            ("1Y", FutureWarning),
+            ("1 Y", FutureWarning),
+            ("1m", FutureWarning),
+            ("1 m", FutureWarning),
+            ("1y", FutureWarning),
+            ("1 y", FutureWarning),
+            ("1 day", None),
+            ("2day", None),
+        ],
+    )
+    def test_unambiguous_timedelta_values(self, val, warning):
+        with tm.assert_produces_warning(warning):
```

Comment:
Could you but the github issue number here as a comment?

Reply:
Done. CI fails due to other tests on master though.